index.js

```diff
@@ -23,7 +23,13 @@ app.use(bodyParser.json())
 app.use(msCorrelation.correlationMiddleware)
 
 const tournamentDataRouter = require('./routers/tournamentDataRouter')
-const matchTeamRouter = require('./routers/matchTeamRouter');
+const matchTeamRouter = require('./routers/matchTeamRouter')
+
+app.use((req, res, next) => {
```

Comment:
This is the exact same change we saw in another PR... See the comments there, You should use the already ready pacakge [cors](https://www.npmjs.com/package/cors)